app/src/main/java/com/developers_recipes/recetario/utils/Constants.kt

```diff
@@ -1,10 +1,13 @@
 package com.developers_recipes.recetario.utils
 
```

Comment:
Would be a constants.kt file and not a class? The "const" keyword, has a static definition by default.

Reply:
its solved